toolbox/include/base/Block.h

```diff
@@ -15,14 +14,15 @@ namespace wbt {
  * Basic abstract class for all the blocks.
  * This class (the whole toolbox in reality) assumes the block represent
  * an instantaneous system (i.e. not a dynamic system).
- * 
+ *
  * You can create a new block by deriving this class and implementing at least
  * all the pure virtual methods.
  *
  * @Note: if you need to implement a block which uses the WBI, you should derive
```

Comment:
Document that if a method returns false, one has to retrieve the singleton Log object